crowdfunding_compassion/controllers/project_controller.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+
+from babel.dates import format_timedelta
+
+from odoo.http import request, route, Controller
+
+
+class ProjectController(Controller):
+    # Demo route used to display demo project
+    @route("/project/", auth="public", website=True)
+    def demo_project_page(self, **kwargs):
+        demo_project = request.env.ref(
+            "crowdfunding_compassion.demo_project_crowdfunding"
+        ).sudo()
+        print(f"Demo project ID: {demo_project.id}")
+        return request.render(
+            "crowdfunding_compassion.project_page",
+            self._prepare_project_values(demo_project, **kwargs),
+        )
+
+    @route(["/project/<int:project_id>"], auth="public", website=True)
```

Comment:
All the routes will start with `/together`. The platform will be called "Together". So here it should be `/together/project/<int...>`